zcash_client_sqlite/src/lib.rs

```diff
@@ -219,6 +238,17 @@ pub struct WalletDb<C, P> {
     params: P,
 }
 
+impl<C: Borrow<rusqlite::Connection>, P: consensus::Parameters> WalletDb<C, P> {
+    /// Returns the account with the given "one-way stable" identifier, if it was created
+    /// by this wallet instance.
+    pub fn get_account_for_uuid(
```

Comment:
While this approach is superficially less intrusive, I believe that it's a significantly worse API from the user perspective to have to retrieve a separate account identifier that they can't serialize every time they want to work with an account.